### PR TITLE
New TXT record because ImprovMX wants it

### PR DESCRIPTION
**Pull request by zvdxc**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] The file's name is all lowercased and alphanumeric. 
- [X] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/mail.zvdxc.json [hunk 1]
```diff
@@ -9,5 +9,6 @@
       "mx1.improvmx.com",
       "mx2.improvmx.com"
     ]
```
Comment (omsenjalia) [inline domains/mail.zvdxc.json:11]: Add a , after this.

**Review by omsenjalia — DISMISSED** [review]

File: domains/mail.zvdxc.json (hunk 1 above)
Comment (zvdxc) [reply domains/mail.zvdxc.json:11]: After the .com?

Comment (VaibhavSys) [inline domains/mail.zvdxc.json:11]:
```suggestion
    ],
```

Comment (VaibhavSys) [reply domains/mail.zvdxc.json]: After ], I did it for you

**Review by VaibhavSys — APPROVED** [review]

File: domains/mail.zvdxc.json (hunk 1 above)
Comment (zvdxc) [reply domains/mail.zvdxc.json]: OK, thanks

**Pull request merged**